tutorials/advanced/transforms_tutorial.py

```diff
@@ -303,12 +316,98 @@
 #
 #     trans = ax.get_xaxis_transform()
 #
+# .. _transforms-fig-scale-dpi:
+#
+# Plotting in physical units
+# ==========================
+#
+# Sometimes we want an object to be a certain physical size on the plot.
+# Here we draw the same circle as above, but in physical units.  If done
+# interactively, you can see that changing the size of the figure does
+# not change the offset of the circle from the lower-left corner,
+# does not change its size, and the circle remains a circle regardless of
+# the aspect ratio of the axes.
+
+fig, ax = plt.subplots(figsize=(5, 4))
+x, y = 10*np.random.rand(2, 1000)
+ax.plot(x, y*10., 'go', alpha=0.2)  # plot some data in data coordinates
+# add a circle in fixed-units
+circ = mpatches.Circle((2.5, 2), 1.0, transform=fig.dpi_scale_trans,
+                      facecolor='blue', alpha=0.75)
+ax.add_patch(circ)
+plt.show()
+
+###############################################################################
+# If we change the figure size...
```

Comment:
I don't like ellipses (...). Either one states what this is meant to be left out or not. In this case, it's probably useful to descibe what happens if you change the figure size, e.g.  
> "If you change the figure size, the circle does not change its absolute position is hence cropped."  

or similar.